blacklistfilter/blacklist_downloader/bl_downloader.py.in

```diff
@@ -124,18 +140,38 @@ class Blacklist:
 
     def download_and_update(self):
         updated = False
+        new_entities = None
 
         try:
             req = requests.get(self.source, timeout=g_conf.socket_timeout)
 
             if req.status_code == 200:
                 data = req.content
 
-                # If CSV, cut the desired column
-                with suppress(AttributeError):
-                    data = self.cut_csv(data)
+                # JSON
+
+                if self.file_format == 'JSON':
+                    try:
+                        data = json.loads(data)  # if not JSON, raises ValueError or TypeError how to catch both?
```

Comment:
This catches both exceptions:
`except (ValueError, TypeError) as e:`

Reply:
Done